src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +565,57 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
+}
+
+model Design_Review {
+  designReviewId   Int                  @id @default(autoincrement())
+  dateScheduled    DateTime
```

Comment:
I'm working on a new datetime policy where we decide to keep track of floating dates (which are the same for all timezones) vs timestamps. For every DateTime field, try adding @db.Date for floating dates and @db.Timestamp(3) for timestamps.